src/main/java/entity/User.java

```diff
@@ -5,11 +5,70 @@
 import org.springframework.cloud.gcp.data.spanner.core.mapping.Column;
 import org.springframework.cloud.gcp.data.spanner.core.mapping.Table;
 
-import java.util.Base64;
 
 @Table(name = "User")
 public class User {
 
+  public static class Builder {
+
+    private Timestamp creationTs;
+    private long userId;
+    private String username;
+    private String password;
+    private String emailId;
+    private String mobileNo;
+    private String picture;
+
+    private Builder() {}
+
+    public Builder creationTs(Timestamp creationTs) {
+      this.creationTs = creationTs;
+      return this;
+    }
+
+    public Builder userId(long userId) {
+      this.userId = userId;
+      return this;
+    }
+
+    public Builder username(String username) {
+      this.username = username;
+      return this;
+    }
+
+    public Builder password(String password) {
+      this.password = password;
+      return this;
+    }
+
+    public Builder emailId(String emailId) {
+      this.emailId = emailId;
+      return this;
+    }
+
+    public Builder mobileNo(String mobileNo) {
+      this.mobileNo = mobileNo;
+      return this;
+    }
+
+    public Builder picture(String picture) {
+      this.picture = picture;
+      return this;
+    }
+
+    public User build() {
+      User user = new User();
+      user.CreationTS = this.creationTs;
+      user.UserID = this.userId;
```

Comment:
So the default is 0?

Reply:
Yes, we will set UserID later in that case.